src/mediaaccessibility.cs

```diff
@@ -45,4 +46,27 @@ interface MAVideoAccommodations {
 		[Field ("kMADimFlashingLightsChangedNotification")]
 		NSString DimFlashingLightsChangedNotification { get; }
 	}
+
+	[TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface MAFlashingLightsProcessorResult {
+		[Export ("surfaceProcessed")]
+		bool SurfaceProcessed { get; }
+
+		[Export ("mitigationLevel")]
+		float MitigationLevel { get; }
+
+		[Export ("intensityLevel")]
+		float IntensityLevel { get; }
+	}
+
+	[TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	[BaseType (typeof (NSObject))]
+	interface MAFlashingLightsProcessor {
+		[Export ("canProcessSurface:")]
+		bool CanProcess (Surface surface);
+
+		[Export ("processSurface:outSurface:timestamp:options:")]
+		MAFlashingLightsProcessorResult Process (Surface inSurface, Surface outSurface, double timestamp, [NullAllowed] NSDictionary<NSString, NSObject> options);
```

Comment:
Probably this is better so we can strong'ify this in the future

```suggestion
		MAFlashingLightsProcessorResult Process (Surface inSurface, Surface outSurface, double timestamp, [NullAllowed] NSDictionary options);
```